ansible_collections/arista/avd/roles/eos_designs/templates/network_services/default-vrf.j2

```diff
@@ -0,0 +1,6 @@
+  redistribute_routes:
+    connected: true
```

Comment:
This will affect the existing fabric behavior. Do we need to redistribute all connected without route-map ? Maybe we could add our new subnets (we have it in the prefix list) to the existing route_map and then exclude those specifically on the peer-group. My point is that we might have other contributors to underlay routing, that we don't know about.

Reply:
Actually, I moved the RM-CONN-2-BGP to IPv4-UNDERLAY-PEER out.

<img width="703" alt="image" src="https://user-images.githubusercontent.com/88734745/154279958-c2a171cc-3709-4be8-a959-00ac0d631631.png">

And redistributed all connected routes here as per the manual.

Reply:
The new connected subnets were added to vrf default route-target export 

<img width="738" alt="image" src="https://user-images.githubusercontent.com/88734745/154280483-482d84ad-0a64-4225-8f8d-181cd063308d.png">

Reply:
I used this [doc](https://docs.google.com/document/d/1VeXddQNdm-k9AtIHWyhL2aB9OZwNuBTpSGbZDjsJpRo/edit#heading=h.4dj21gsbghfz) and the one provided by you for reference.

Reply:
I have tested the changes you mentioned: 

<img width="506" alt="image" src="https://user-images.githubusercontent.com/88734745/154290729-a9bbafd0-b9d7-42b4-952d-ae31b4be437b.png">

<img width="662" alt="image" src="https://user-images.githubusercontent.com/88734745/154290836-fa39f0cc-5fce-4a85-848a-9b36db83cf2c.png">

<img width="375" alt="image" src="https://user-images.githubusercontent.com/88734745/154291655-e8dfcd5a-48a3-4eab-87f4-c1efb6783186.png">

Reply:
I tested that with customer. It seems like we have a "double filter" for the redistribution. 
Basically, if you use the "redistribute connected" command with a route-map, no other networks can be considered later in the VRF to be announced out of these allowed in the route-map.
This is the reason you filter only to spines direction, and you can't add a route-map to the redistribute connected.

May a general redistribute connected with a general route-map and then a more specific one towards the spines to filter underlay? This is not documented, I am not sure if we should go down this path.

Reply:
Thank you for the clarification.